rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/execution/Configuration.kt

```diff
@@ -261,7 +261,7 @@ data class JarikoCallback(
      * Default implementation provides a simple println with the name of the mock statement.
      * @param mockStatement The mock statement
      */
-    var onMockStatement: ((mockStatement: MockStatement) -> Unit) = { System.err.println("Executing mock statement: ${it.javaClass.simpleName}") },
+    var onMockStatement: ((mockStatement: MockStatement) -> Unit) = { System.err.println("Executing mock statement: ${it.javaClass.simpleName} ${if (it is Statement) "at " + it.position + "." else "." }") },
```

Comment:
`it.position` is absolute, so to avoid misinterpretation, in the log replace `"at it.position"` with `"at absolute position: it.position"`.

Reply:
Done